Add 2.2 RNs (#31737)
openshift/openshift-docs: modules/developer-cli-odo-adding-storage-to-a-specific-container.adoc

// Module included in the following assemblies:
//
//creating_and_deploying_applications_with_odo/working-with-storage.adoc

[id="adding-storage-to-a-specific-container_{context}"]
= Adding storage to a specific container

> If your devfile has multiple containers, you can specify to which container you want the storage to attach by using the `--container` flag.

Reviewer: ```suggestion
If your devfile has multiple containers, you can use the `--container` flag to specify the container you want the storage to.
```
After being punished so often for the pointless grammar rule from LATIN(!) forbidding prepositions at the ends of sentences, I felt joy when I found the following IBM SG rule: 
`Use a preposition at the end of a sentence to avoid an awkward or stilted construction.
Awkward: Click the item for which you want to search.
Improved: Click the item that you want to search for.`
